suite-native/device/src/deviceThunks.ts

```diff
@@ -0,0 +1,41 @@
+import { pipe, A } from '@mobily/ts-belt';
+
+import { createThunk } from '@suite-common/redux-utils';
+import {
+    selectDevices,
+    selectAccounts,
+    accountsActions,
+    selectDiscovery,
+    discoveryActions,
+    PORTFOLIO_TRACKER_DEVICE_STATE,
+} from '@suite-common/wallet-core';
+import { Account } from '@suite-common/wallet-types';
+
+const actionPrefix = '@suite-native/device';
+
+export const wipeDisconnectedDevicesDataThunk = createThunk(
+    `${actionPrefix}/wipeDisconnectedDevicesDataThunk`,
+    (_, { getState, dispatch }) => {
+        // TODO: When we enable remember mode, also `disconnected remembered` (device.remember === true) devices should stay untouched.
```

Comment:
It should be pretty easy to add check right now, maybe better do it now and save some time debugging in future?

Reply:
I checked again and the check for `remember` prop would be useless. It is already checked in the `disconnect` device action. If it is not remembered, it is deleted from the state. Otherwise it stays there. 

For this thunk is important if the device is in the state or not, so the `disconnect` action handles this already. 

--> I've removed the TODO comment, since it was useless.